.gitattributes

```diff
@@ -0,0 +1 @@
+requirements.txt merge=union
```

Comment:
I think pip is going to complain when there are duplicate requirements, but these can then be resolved manually if and when it happens.

Reply:
Yes, and it will complain only if 2 lines have a different version's requirement.